packages/components/inputs/radio-input/src/radio-option.tsx

```diff
@@ -17,7 +18,10 @@ import {
   getLabelStyles,
   getContainerStyles,
   LabelTextWrapper,
+  RadioOptionsWrapper,
+  AdditionalTextWrapper,
 } from './radio-option.styles';
+import Spacings from '../../../../../presets/spacings';
```

Comment:
We need an absolute `import` statement here.

```suggestion
import Spacings from '@commercetools-uikit/spacings';
```